tests/test_workdir.py

```diff
@@ -57,12 +59,53 @@ def test_directory_structure_generated(self, spatial_directory_list):
             assert path.exists()
 
 
-class TestCleanDirectoryTree:
-    """Tests for `clean_directory_tree()`."""
+class TestCleanFiles:
+    """Tests for `clean_directory_tree()` and `clean_cwd_logfiles)."""
 
-    @pytest.mark.parametrize("test_path", [Path("runs"), Path("src")])
-    def test_clean_directory_tree(self, test_path):
-        """Success case: directory tree does not exist after clean."""
-        test_path.mkdir()
-        clean_directory_tree()
-        assert not test_path.exists()
+    @pytest.fixture(autouse=True)
+    def set_internal_cwd(self, monkeypatch):
+        """Sets internal.CWD to pytest's working directory."""
+        monkeypatch.setattr(internal, "CWD", Path.cwd())
+
+    @pytest.fixture()
+    def src_path(self) -> Path:
+        return Path("src")
+
+    @pytest.fixture()
+    def runs_path(self) -> Path:
+        return Path("runs")
+
+    @pytest.fixture(params=["rev_number-0.log", "rev_number-200.log"])
+    def revision_log_file(self, request) -> Path:
+        return Path(request.param)
+
+    @pytest.fixture(
+        params=["benchmark_cable_qsub.sh.o21871", "benchmark_cable_qsub.sh"]
+    )
+    def pbs_job_file(self, request) -> Path:
+        return Path(request.param)
+
+    def test_clean_realisation_files(
+        self, src_path: Path, tmp_path: Path, revision_log_file: Path
```

Comment:
I can't see where `tmp_path` is defined.

Reply:
`tmp_path` is an inbuilt fixture by `pytest` (which would do the necessary cleanup after the test)

Reply:
Right, sorry I had forgotten the name of it.